Dockerfile

```diff
@@ -1,16 +1,22 @@
+# ubuntu:14.04 -- https://hub.docker.com/_/ubuntu/
+# |==> phusion/baseimage:0.9.17 -- https://goo.gl/ZLt61q
+#      |==> phusion/passenger-ruby22:0.9.15 -- https://goo.gl/xsnWOP
+#           |==> HERE
 FROM phusion/passenger-ruby22:0.9.15
 
-EXPOSE 80
 ENV APP_HOME=/home/app/pact_broker
-CMD ["/sbin/my_init"]
+
 RUN rm -f /etc/service/nginx/down
 RUN rm /etc/nginx/sites-enabled/default
-ADD container /
 
-ADD pact_broker/Gemfile $APP_HOME/
-ADD pact_broker/Gemfile.lock $APP_HOME/
+ADD container /
+ADD pact_broker $APP_HOME
 RUN chown -R app:app $APP_HOME
-RUN su app -c "cd $APP_HOME && bundle install --deployment --without='development test'"
 
```

Comment:
Separating the files is deliberate, to allow changes to be made to later files without having to rebuild all the layers.